templates/mission_edit.html

```diff
@@ -118,46 +124,100 @@ <h1>Edit Mission</h1>
 					</label>
 				</td>
 				<td>
-					<input type="url" name="drive_url" value="{{ mission.drive_url or '' }}" />
+					<input type="url" name="drive_url" value="{{ mission.drive_url or '' }}" placeholder="https://drive.google.com/drive/folders/XXXXXXXX...XXXXXXXX" pattern="https://drive.google.com/drive/folders/[A-Za-z0-9_\-]+" />
+				</td>
+			</tr>
+			<tr>
+				<td></td>
+				<td>
+					<hr />
 				</td>
 			</tr>
 			<tr>
 				<td>
-					<label for="fb_url">Facebook event</label>
+					<label for="intro_url">Intro. presentation</label>
 				</td>
 				<td>
-					<input type="url" name="fb_url" value="{{ mission.fb_url or '' }}" />
+					<input type="url" name="intro_url" value="{{ mission.intro_url or '' }}" placeholder="https://docs.google.com/presentation/d/XXXXXXXX...XXXXXXXX/edit" pattern="https://docs.google.com/presentation/d/[A-Za-z0-9_\-]+/edit" />
 				</td>
 			</tr>
 			<tr>
 				<td>
-					<label for="gplus_url">Google+ event</label>
+					<label for="pres_url">Mission presentation</label>
 				</td>
 				<td>
-					<input type="url" name="gplus_url" value="{{ mission.gplus_url or '' }}" />
+					<input type="url" name="pres_url" value="{{ mission.pres_url or '' }}" placeholder="https://docs.google.com/presentation/d/XXXXXXXX...XXXXXXXX/edit" />
+					<!-- No validation on this field because it could be Prezi, PowerPoint Online, etc. -->
 				</td>
 			</tr>
 			<tr>
 				<td>
-					<label for="the_link_url">The Link event</label>
+					<label for="sign_in_url">Sign-in form</label>
+				</td>
+				<td>
+					<input type="url" name="sign_in_url" value="{{ mission.sign_in_url or '' }}" placeholder="https://docs.google.com/forms/d/XXXXXXXX...XXXXXXXX/viewform" pattern="https://docs.google.com/forms/d/[A-Za-z0-9_\-]+/viewform" />
+				</td>
+			</tr>
+			<tr>
+				<td></td>
+				<td>
+					<hr />
+				</td>
+			</tr>
+			<tr>
+				<td>
+					<label for="fb_url">Facebook event</label>
 				</td>
 				<td>
-					<input type="url" name="the_link_url" value="{{ mission.the_link_url or '' }}" />
+					<input type="url" name="fb_url" value="{{ mission.fb_url or '' }}" placeholder="https://www.facebook.com/events/XXXXXXXX...XXXXXXXX" pattern="https://www.facebook.com/events/[0-9]+" />
```

Comment:
The facebook event I tested also had a trailing '/' and failed this check, should have been accepted